setup.py

```diff
@@ -98,7 +98,7 @@ def extra(self):
         'dataclasses>=0.6;python_version<"3.7"'
     ],
     extras_require={
-        'ujson': ['ujson>=1.35'],
+        'json': ['orjson>=2.0.6,<3;platform_python_implementation=="CPython"'],
```

Comment:
I haven't tested this on PyPy, but the intent is to allow users to unconditionally install `pydantic[json]` without an issue.

Reply:
> I haven't tested this on PyPy, but the intent is to allow users to unconditionally install `pydantic[json]` without an issue.

I think this might be confusing -- it might make someone think that for *any* json support they need to install `pydantic[json]`. I would vote to have it as `'orjson': ['orjson>=2.0.6,<3;platform_python_implementation=="CPython"']`

Reply:
Ok. The intent was the target won't change if the implementation does. I've changed it to orjson.